pkg/parcacol/normalizer.go

```diff
@@ -188,11 +190,11 @@ func LabelsFromSample(takenLabelNames map[string]string, stringTable []string, p
 	resLabels := map[string]string{}
 	for _, labelName := range labelNames {
 		resLabelName := labelName
-		if _, ok := takenLabelNames[resLabelName]; ok {
-			resLabelName = "exported_" + resLabelName
+		if _, ok := takenLabels[resLabelName]; ok {
+			resLabelName = model.ExportedLabelPrefix + resLabelName
 		}
 		if _, ok := resLabels[resLabelName]; ok {
```

Comment:
Why is there another check done after the collision was handled? 🤔 

Is that in case I define labels `instance` and `exported_instance` in my pprof_labels when `instance` is already part of the labels?